go.mod

```diff
@@ -26,3 +26,5 @@ require (
 	golang.org/x/crypto v0.0.0-20210322153248-0c34fe9e7dc2
 	gopkg.in/yaml.v2 v2.4.0
 )
+
+replace golang.org/x/crypto/ssh/terminal => golang.org/x/term v0.0.0-20210503060354-a79de5458b56
```

Comment:
this hasn't fixed this it seems https://github.com/srl-labs/containerlab/pull/416/checks?check_run_id=2546702135#step:4:26

Reply:
why not swapping the pkg import to x/term though?

Reply:
it's an external dependency, from mysocket-io

Reply:
It's not. I don't use mysocket API, rather shelling out and using it's api
Here is the term thing I use to capture the password from the terminal 

https://github.com/srl-labs/containerlab/blob/169c8bdb33d78a6bb5006d568dbc396561af8eb9/cmd/mysocketio.go#L14